samples/server/petstore/kotlin-springboot/src/main/kotlin/org/openapitools/model/Pet.kt

```diff
@@ -20,22 +20,22 @@ import io.swagger.annotations.ApiModelProperty
  */
 data class Pet (
 
-        @get:NotNull 
+    @get:NotNull 
         @ApiModelProperty(example = "doggie", required = true, value = "")
-        @JsonProperty("name") val name: kotlin.String,
+        @JsonProperty("name") val name: String,
 
-        @get:NotNull 
+    @get:NotNull 
```

Comment:
Formatting here looks a little inconsistent

Reply:
Fixed that now